app/helpers/dot_helper.rb

```diff
@@ -8,9 +8,15 @@ def render_agents_diagram(agents)
         } rescue false)
       decorate_svg(svg, agents).html_safe
     else
-      tag('img', src: URI('https://chart.googleapis.com/chart').tap { |uri|
-            uri.query = URI.encode_www_form(cht: 'gv', chl: agents_dot(agents))
-          })
+	uriquery=URI.encode_www_form(cht: 'gv', chl: agents_dot(agents))
+	#Get query maximum length should be under 2048 bytes with including "chart?" of google chart request url
+	if uriquery.length > 2042
```

Comment:
Thanks @HyeongBaeYu!

Can you please fix the indentation so that it's always 2 spaces?